### PR TITLE
Update library usage article for v4

### LINKED ISSUE #40: Update Library-Usage.md
### Summary
 
This article still mentions PCL which we do not use anymore. We could also mention nuget package management using the default tooling here as well.

### PR DESCRIPTION
**Pull request by Rick-van-Dam**

### CODE REVIEW

File: pages/v4/Library-Usage.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ In general, when you want to use a (managed) library in your game plugin, you wi
 
 ![](../img/LibraryUsage/AddReference.png)
 
-If you're familiar with it, you can also use the Visual Studio integrated version of NuGet and add your dependency as a package - though it is advised not to do that for Duality plugin packages specifically, since they are already handled by Duality itself.
+You can also use the Visual Studio integrated version of NuGet and add your dependency as a package. In fact since v4 this is now the default way duality manages packages.
```
Comment (SirePi) [inline pages/v4/Library-Usage.md]: Duality, capital

File: pages/v4/Library-Usage.md [hunk 2]
```diff
@@ -23,19 +23,19 @@ There are various kinds of libraries that you can use with Duality, some of them
...
+You can reference managed libraries that are not Duality plugins as well, same procedure as above. In most cases, that's all you need to know.
 
-### Portability and (non-) Portable Class Libraries
+### Portability and (non-) .NET Standard Class Libraries
 
-By default, Duality game plugins are [Portable Class Libraries](Framework-Structure.md), which means that they limit themselves to a subset of the .Net Framework that is known to be portable across a range of platforms. To keep this contract intact, you can only reference libraries that use a similar or smaller subset of .Net, thus retaining portability.
+By default, Duality game plugins are [.NET Standard Class Libraries](Framework-Structure.md), which means that they limit themselves to a subset of .NET that is known to be portable across all .NET implementations. To keep this contract intact, you can only reference libraries that use a similar or smaller subset of .NET, thus retaining portability. See the [microsoft .NET Standard docs page](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) for more info about .NET Standard.
```
Comment (SirePi) [inline pages/v4/Library-Usage.md:36]: Would it be necessary to specify that is .NET Standard 2.0?

**Review by SirePi — CHANGES_REQUESTED** [review]

File: pages/v4/Library-Usage.md (hunk 2 above)
Comment (Rick-van-Dam) [reply pages/v4/Library-Usage.md:36]: No, not in this article but in https://docs.duality2d.net/pages/v4/Framework-Structure.html

Comment (Rick-van-Dam) [reply pages/v4/Library-Usage.md:36]: There is already a link to that article and repeating these details means we have to keep them in sync as well

**Review by SirePi — APPROVED** [review]

**Pull request merged**